src/burningglass/01_data_profiling_complete.Rmd

```diff
@@ -38,6 +46,10 @@ pers <- DBI::dbGetQuery(con, "
 )
 
 DBI::dbDisconnect(con)
+
+uszips <- read_csv("~/git/dspg20career/src/burningglass/uszips.csv")%>%
```

Comment:
That assumes `dspg20career` is cloned at `~/git`. I would be better to have something like
```
library(here) # If using an RProject it will set the base path to the base path of the repository
uszips <- read_csv(file = here("src", "burningglass", "uszips.csv"))
```